Cargo.toml

```diff
@@ -4,6 +4,7 @@ cargo-features = ["resolver"]
 resolver = "2"
 members = [
     "full-service",
+    "mirror",
```

Comment:
All of this seems standalone, but perhaps this should be an optional feature? 🤔

Reply:
not sure what you mean by this?

Reply:
incase someone wants to build FS and isn't going to use the mirror, but not a big deal since it's probably not much extra overhead

Reply:
that's what the -p flag is for during build, so if you wanted to just specify full-service you would do cargo build -p mc-full-service

Reply:
O